src/plugin.js

```diff
@@ -131,7 +131,7 @@ function contextmenuUI(options) {
     excludeElements
   };
 
-  options = videojs.mergeOptions(defaults, options);
+  options = videojs.obj.merge(defaults, options);
```

Comment:
Do we want to drop v7 support or fall back to `videojs.mergeOptions`?

Reply:
That's a good point! Since the language of the deprecation warnings is `...is deprecated and will be removed in 9.0`, it does seem like we should provide a fallback.

Reply:
Thinking on this, I'm not actually sure why we would want to provide these kinds of fallbacks for code within the plugin itself, especially if we're specifying videojs 8 as a dependency, as part of this change. We know `videojs.obj.merge` is available in v8.

Reply:
Yeah, as above, I think this should be a major version of this plugin. In that case, we can drop v7 support.